toolchain/syno-qoriq-6.2.4/Makefile

```diff
@@ -9,5 +9,16 @@ TC_DIST_SITE_PATH = PowerPC%20QorIQ%20Linux%202.6.32
 TC_TARGET = powerpc-e500v2-linux-gnuspe
 TC_SYSROOT = $(TC_TARGET)/sysroot
 TC_EXTRA_CFLAGS = -mcpu=8548 -mhard-float -mfloat-gprs=double
+TC_RUSTFLAGS = -C target-cpu=e500
+
+POST_FIX_TARGET = qoriq_post_fix_target
 
 include ../../mk/spksrc.tc.mk
+
+.PHONY: qoriq_post_fix_target
+qoriq_post_fix_target:
+	@cd $(WORK_DIR)/$(TC_TARGET)/bin ; \
+	for gnutool in $$(ls -1); do \
+	  [ ! -h "powerpc-linux-gnuspe-$${gnutool##*-}" ] && ln -s $${gnutool} "powerpc-linux-gnuspe-$${gnutool##*-}" ; \
+	  [ ! -h "powerpc-unknown-linux-gnuspe-$${gnutool##*-}" ] && ln -s $${gnutool} "powerpc-unknown-linux-gnuspe-$${gnutool##*-}" ; \
```

Comment:
please use "-L" and avoid the use of "-h" 
regarding this article:
https://stackoverflow.com/questions/1997666/difference-between-test-h-and-test-l
"-h" is legacy version that might be removed some day.

or probably omit the test whether the links not already exist and force the creation (ln -sf )

Reply:
done